src/rpc/bobtail.cpp

```diff
@@ -0,0 +1,245 @@
+// Copyright (c) 2020 The Bitcoin Unlimited developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include "amount.h"
+#include "blockrelay/netdeltablocks.h"
+#include "blockstorage/blockstorage.h"
+#include "bobtail/bobtail.h"
+#include "bobtail/dag.h"
+#include "bobtail/miner.h"
+#include "bobtail/validation.h"
+#include "chain.h"
+#include "chainparams.h"
+#include "consensus/consensus.h"
+#include "consensus/params.h"
+#include "consensus/validation.h"
+#include "core_io.h"
+#include "dstencode.h"
+#include "init.h"
+#include "main.h"
+#include "net.h"
+#include "parallel.h"
+#include "pow.h"
+#include "rpc/server.h"
+#include "txadmission.h"
+#include "txmempool.h"
+#include "ui_interface.h"
+#include "util.h"
+#include "utilstrencodings.h"
+#include "validationinterface.h"
+
+#include <cstdlib>
+#include <stdint.h>
+
+#include <boost/assign/list_of.hpp>
+#include <boost/shared_ptr.hpp>
+
+extern CBobtailDagSet bobtailDagSet;
+
+UniValue generateBobtailBlocks(boost::shared_ptr<CReserveScript> coinbaseScript,
+    int nGenerate,
+    uint64_t nMaxTries,
+    bool keepScript,
+    int weak_mode)
```

Comment:
I believe we should always be in "weak mode". So probably can remove this param.

Reply:
good catch, fixed